packages/keyring-eth-ledger-bridge/src/ledger-iframe-bridge.ts

```diff
@@ -338,7 +319,7 @@ export class LedgerIframeBridge
 
   async #sendMessage<TAction extends IFrameMessageAction>(
     message: IFrameMessage<TAction>,
-    timeout = IFRAME_MESSAGE_TIMEOUT,
+    { timeout }: { timeout?: number } = {},
```

Comment:
Matching the signature of the `#sendMessage` method we are adding on the MV3 extension offscreen iframe bridge: https://github.com/MetaMask/metamask-extension/pull/31766